frame/nomination-pools/benchmarking/src/lib.rs

```diff
@@ -262,16 +287,17 @@ frame_benchmarking::benchmarks! {
 
 	}: bond_extra(RuntimeOrigin::Signed(scenario.creator1.clone()), BondExtra::Rewards)
 	verify {
+		 // commission of 50% deducted here.
 		assert!(
 			T::Staking::active_stake(&scenario.origin1).unwrap() >=
-			scenario.dest_weight
+			scenario.dest_weight / 2u32.into()
 		);
 	}
 
 	claim_payout {
 		let origin_weight = Pools::<T>::depositor_min_bond() * 2u32.into();
 		let ed = CurrencyOf::<T>::minimum_balance();
-		let (depositor, pool_account) = create_pool_account::<T>(0, origin_weight);
+		let (depositor, pool_account) = create_pool_account::<T>(0, origin_weight, None);
```

Comment:
This is actually he benchmark where you would have to set the commission, because the worse-case of a payout is one with commission. In all the rest, I would create the pool without commission because having it is of no importance in e.g. `create`, `bond` etc.